Add uninstall instructions to RUM auto instrumentation server-side (#28703)
DataDog/documentation: content/en/real_user_monitoring/browser/setup/server/ibm.md

[2]: /agent/
[3]: https://httpd.apache.org/modules/
[4]: https://rum-auto-instrumentation.s3.amazonaws.com/httpd/latest/mod_datadog-amd64.zip
[5]: https://app.datadoghq.com/rum/list

## Uninstall

> To manually remove the RUM Auto-Instrumentation:

Author: ```suggestion
To manually remove RUM from your auto-instrumented web server:
```

> While automatic uninstallation will be available soon, you can manually remove the RUM Auto-Instrumentation by following these steps.

Reviewer: ```suggestion
To manually remove the RUM Auto-Instrumentation:
```

> 1. Locate your IBM HTTP Server binary, it may be one of `httpd`, `apachectl`, `apache2` or `apache2ctl`. The following steps use `httpd` as an example.

Reviewer: ```suggestion
```

> 1. Locate your IBM HTTP Server binary, it may be one of `httpd`, `apachectl`, `apache2` or `apache2ctl`. `httpd` will be used for illustration purposes.

Reviewer: ```suggestion
1. Locate your IBM HTTP Server binary, it may be one of `httpd`, `apachectl`, `apache2` or `apache2ctl`. The following steps use `httpd` as an example.
```

> 2. Locate the IBM HTTP Server configuration file by running `httpd -V`. For example: `/usr/local/apache2/conf/httpd.conf`.

Reviewer: ```suggestion
1. Locate the IBM HTTP server (`httpd`) configuration file by running `httpd -V`. Depending on the Linux distribution used, this binary file could be named `http`, `apachectl`, `apache2` or `apache2ctl`. The following steps use `httpd` as an example. In this instance, the file location could be: `/usr/local/apache2/conf/httpd.conf`.
```

> 2. Locate the IBM HTTP Server configuration file by running `httpd -V`. Example: `/usr/local/apache2/conf/httpd.conf`.

Reviewer: ```suggestion
2. Locate the IBM HTTP Server configuration file by running `httpd -V`. For example: `/usr/local/apache2/conf/httpd.conf`.
```

> 3. At the end of the file, remove the line: `Include /opt/datadog-httpd/datadog.conf`.

Reviewer: ```suggestion
2. At the end of the file, remove the line: `Include /opt/datadog-httpd/datadog.conf`.
```

> 4. Delete the directory `/opt/datadog-httpd/` and all of its contents.

Reviewer: ```suggestion
3. Delete the directory `/opt/datadog-httpd/` and all of its contents.
```

> 5. Restart or reload the IBM HTTP Server.

Reviewer: ```suggestion
4. Restart or reload the IBM HTTP Server.
```

> 4. Delete the directory `/opt/datadog-httpd/` and all its contents.

Reviewer: ```suggestion
4. Delete the directory `/opt/datadog-httpd/` and all of its contents.
```

> 5. Finally, restart or reload IBM HTTP Server.

Reviewer: ```suggestion
5. Restart or reload the IBM HTTP Server.
```
